### ISSUE TITLE
Trying to require Beaker in a Phoenix project 

### ISSUE THREAD

**Issue by rockerBOO** [hahuang65/beaker#4]
Wanted to add this to my current phoenix project, but it is not working correctly. I don't have a :phoenix compiler, but not sure why I’d need that. I am not sure I want the Phoenix integration though.

My `mix.exs` applications list
```elixir
     applications: [:phoenix, :phoenix_html, :cowboy,
                    :logger, :phoenix_ecto, :postgrex,
                    :quantum,  :beaker, :httpoison,
                    :exredis, :mongodb]]
```

Error when trying to compile project
```elixir
** (RuntimeError) could not load template_engines configuration for Phoenix. Please ensure you have listed :phoenix under :applications in your mix.exs file and have enabled the :phoenix compiler under :compilers
    lib/phoenix/template.ex:238: Phoenix.Template.raw_config/1
    lib/phoenix/template.ex:228: Phoenix.Template.compiled_engines/0
    lib/phoenix/template.ex:299: Phoenix.Template.find_all/1
    expanding macro: Phoenix.Template.__before_compile__/1
    lib/web/views/layout_view.ex:2: Beaker.LayoutView (module)
```

**Comment by hahuang65** [comment]
Sorry for the late response, I've been quite busy.

This is a bug... `Phoenix.HTML` will be required for Phoenix integration, which will be release in 0.0.4
However, this is only if you want to have a page that visualizes your counters and gauges in Phoenix. Otherwise it's an optional dependency.

I'm about to release 0.0.4 in a few minutes, can you try updating to that and let me know if that fixes it?

**Comment by hahuang65** [comment]
This should be fixed in 0.0.4. Feel free to reopen if it didn't.
